dbms/src/Flash/tests/gtest_interpreter.cpp

```diff
@@ -93,6 +93,11 @@ CATCH
 TEST_F(InterpreterExecuteTest, MultipleQueryBlockWithSource)
 try
 {
+    context.scan("test_db", "test_table_1")
```

Comment:
remove it

Reply:
why?

Reply:
> why?

I accidently add the code, it is not tested.